api/folders/controllers/folders.js

```diff
@@ -4,28 +4,51 @@ const { sanitizeEntity } = require("strapi-utils");
 
 module.exports = {
   async tree(ctx) {
-    let entities = await strapi.services.folders.find();
-    entities = entities.map((entity) =>
+    let foldersEntities = await strapi.services.folders.find();
+
+    foldersEntities = foldersEntities.map((entity) =>
       sanitizeEntity(entity, { model: strapi.models.folders })
     );
+
+    const { tag_id_in } = ctx.query;
+    let bookmarksFiltered = [];
+    if (tag_id_in) {
+      const tags = eval(tag_id_in);
```

Comment:
`tag_id_in.split(",");`

Reply:
I've removed the [] from the format; so it will be ?tag_id_in=2,3

Reply:
Done